impl/src/bitfield/expand.rs

```diff
@@ -387,6 +388,7 @@ impl BitfieldStruct {
                     }
                 }
 
+                #[allow(unused_braces)]
```

Comment:
please write a short comment describing why we are silencing this warning here (2)

Reply:
Done